scripts/classification/FastTextGluon.py

```diff
@@ -0,0 +1,265 @@
+from collections import Counter
+import itertools
+import numpy as np
+import re
+import gluonnlp
+from collections import Counter
+# import dependencies
+import mxnet as mx
+import mxnet.ndarray as F
+import mxnet.gluon as gluon
+from mxnet.gluon import nn
+from mxnet import autograd
+from mxnet.contrib import text
+from mxnet import nd, autograd, gluon
+from mxnet.gluon import nn, rnn, Block, HybridBlock
+import re
+from sklearn.preprocessing import LabelEncoder
+from sklearn.preprocessing import LabelBinarizer
+import gluonnlp.data.batchify as btf
+import argparse
+import evaluation
+import logging
```

Comment:
Thanks for contributing!
There are several redundant or unused imports, such as autograd. Could you resolve it?